ci-operator/step-registry/openshift-qe/installer/bm/day2/perf-profile/rds-core/openshift-qe-installer-bm-day2-perf-profile-rds-core-commands.sh

```diff
@@ -60,7 +64,7 @@ spec:
     userLevelNetworking: false
 EOF
 
-sleep 60
+oc adm wait-for-stable-cluster --minimum-stable-period=2m --timeout=20m
 kubectl wait --for jsonpath='{.status.updatedMachineCount}'="$(oc get node --no-headers -l node-role.kubernetes.io/worker= | wc -l)" --timeout=30m mcp worker
```

Comment:
I think the wait for machineset should go before the wait for cluster operators health.
Also, please just for consistency parametrize the same variables as in: https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift-qe/cluster-health/openshift-qe-cluster-health-ref.yaml

Reply:
cluster health doesn't have anything to do with machineset wait right. Do you really think the order matters? I did n't parameterize the health check times as they are used in this single step as an adhoc thing and did not see a value in doing so.

Reply:
The machineset wait will always finish before the CO check, I just think the order doesn't make sense.

Reply:
okay. Updated as suggested.